html2/tokenizer.cpp

```diff
@@ -4,6 +4,8 @@
 
 #include "html2/tokenizer.h"
 
+#include "util/string.h"
+
 #include <spdlog/spdlog.h>
 
 #include <cctype>
```

Comment:
I'm pretty sure `std::tolower` was the only thing from cctype, so this header can be nuked now!

Reply:
Fixed!